libraries/lib-stretching-sequence/AudioSegmentFactory.cpp

```diff
@@ -28,15 +29,16 @@ AudioSegmentFactory::AudioSegmentFactory(
 
 std::vector<std::shared_ptr<AudioSegment>>
 AudioSegmentFactory::CreateAudioSegmentSequence(
-   double playbackStartTime, PlaybackDirection direction) const
+   double playbackStartTime, PlaybackDirection direction)
 {
+   mPropagators.clear();
```

Comment:
I think `mPropagators` never gets size more than 1, so why a vector?

Reply:
It's outdated now, through the use of `Observer`.